### PR TITLE
Microsoft.VisualStudioCode.Insiders_v1.81.0

### PR DESCRIPTION
**Pull request by lszomoru**
Microsoft.VisualStudioCode.Insiders_v1.81.0
 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111739&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Commands:
+  - code-insiders
+InstallerType: inno
+InstallerSwitches:
+  Custom: /mergetasks=!runcode
+Installers:
+  - Architecture: arm64
+    Scope: machine
+    InstallerUrl: 'https://az764295.vo.msecnd.net/insider/5150ef0ce77ef5516e55af4b8272f25907b55953/VSCodeSetup-arm64-1.81.0-insider.exe'
+    InstallerSha256: bcb1b734dc67ea1eaf7f912408fcd6abde1ac9dcdbecca62cd67af9dbd98c016
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml:14]:
```suggestion
    InstallerSha256: 1ac394c51486d6197a067f4306f2bf5d700114ff722050a9667ae044775dfe05
```

File: manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    InstallerUrl: 'https://az764295.vo.msecnd.net/insider/5150ef0ce77ef5516e55af4b8272f25907b55953/VSCodeSetup-arm64-1.81.0-insider.exe'
+    InstallerSha256: 1ac394c51486d6197a067f4306f2bf5d700114ff722050a9667ae044775dfe05
+  - Architecture: x64
+    Scope: machine
+    InstallerUrl: 'https://az764295.vo.msecnd.net/insider/5150ef0ce77ef5516e55af4b8272f25907b55953/VSCodeSetup-x64-1.81.0-insider.exe'
+    InstallerSha256: f39f633f78a8f30a3f2ff3c53d0723f9806b0823ddb83c174d6330ac018b155b
+  - Architecture: x64
+    Scope: user
+    InstallerUrl: 'https://az764295.vo.msecnd.net/insider/5150ef0ce77ef5516e55af4b8272f25907b55953/VSCodeSetup-x64-1.81.0-insider.exe'
+    InstallerSha256: e82c282cff9acf956aef9b99f79e84a0c718b02091693860114572720967fe22
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml:26]:
```suggestion
    InstallerSha256: F39F633F78A8F30A3F2FF3C53D0723F9806B0823DDB83C174D6330AC018B155B
```

File: manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - Architecture: x86
+    Scope: machine
+    InstallerUrl: 'https://az764295.vo.msecnd.net/insider/5150ef0ce77ef5516e55af4b8272f25907b55953/VSCodeSetup-ia32-1.81.0-insider.exe'
+    InstallerSha256: eba8731563cac1af95873da35ba8f348ec98b1d5d0b265e2748912bd616d900a
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml:30]:
```suggestion
    InstallerSha256: EBA8731563CAC1AF95873DA35BA8F348EC98B1D5D0B265E2748912BD616D900A
```

File: manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml (hunk 1 above)
Comment (CoolPlayLin) [inline manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml:14]:
```suggestion
    InstallerSha256: 1ac394c51486d6197a067f4306f2bf5d700114ff722050a9667ae044775dfe05
```

File: manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  - Architecture: x86
+    Scope: user
+    InstallerUrl: 'https://az764295.vo.msecnd.net/insider/5150ef0ce77ef5516e55af4b8272f25907b55953/VSCodeSetup-ia32-1.81.0-insider.exe'
+    InstallerSha256: 0b1cb303c9575e48b778f46b9f41bf7939d045a5e7cec7ffe451e156e8478bb0
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml:34]:
```suggestion
    InstallerSha256: EBA8731563CAC1AF95873DA35BA8F348EC98B1D5D0B265E2748912BD616D900A
```

File: manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml (hunk 2 above)
Comment (CoolPlayLin) [inline manifests/m/Microsoft/VisualStudioCode/Insiders/1.81.0/Microsoft.VisualStudioCode.Insiders.installer.yaml:26]:
```suggestion
    InstallerSha256: F39F633F78A8F30A3F2FF3C53D0723F9806B0823DDB83C174D6330AC018B155B
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**